test/cypress/stubs/service-stubs.js

```diff
@@ -8,9 +8,15 @@ function getServiceSuccess (opts) {
     gateway_account_ids: [opts.gatewayAccountId],
     service_name: opts.serviceName,
     external_id: opts.serviceExternalId,
-    organisationName: opts.organisationName
+    organisationName: opts.organisationName,
+    custom_branding: opts.customBranding ? {
+      image_url: opts.customBranding.imageUrl,
+      css_url: opts.customBranding.cssUrl
+    } : null
   }
 
+  console.log('== fixture opts: ', fixtureOpts)
```

Comment:
Is this necessary for the Cypress output?

Reply:
Good spot - just removed it.